recipes/bxdecay0/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "1.1.2" %}
+
+package:
+  name: bxdecay0
+  version: {{ version }}
+
+source:
+  url: https://github.com/BxCppDev/bxdecay0/archive/{{ version }}.tar.gz
+  sha256: 3c0a621f78629125159e75842bd8c8250698eca653804a37a1e6b4581c5d6fa1
+
+build:
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('bxdecay0', max_pin='x.x') }}
```

Comment:
Do you have an upstream documentation ensuring that there are not ABI breakages on patch releases?

Reply:
I checked again, and I am even not entirely sure if they cleanly follow semantic versioning from a feature perspective. For safety I now also included the patch version.